pkg/controllers/disruption/controller.go

```diff
@@ -250,7 +250,8 @@ func (c *Controller) logAbnormalRuns(ctx context.Context) {
 func (c *Controller) logInvalidBudgets(ctx context.Context) {
```

Comment:
Discussed offline. I think the method signature should just be `ValdiateBudgets() error` so that we are less likely to miss error returns like this